client/src/webview/TablePropertiesViewer.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright © 2025, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import "./TablePropertiesViewer.css";
+
+function showTab(tabName: string, clickedTab?: HTMLElement): void {
+  const contents = document.querySelectorAll(".tab-content");
+  contents.forEach((content) => content.classList.remove("active"));
+
+  const tabs = document.querySelectorAll(".tab");
+  tabs.forEach((tab) => tab.classList.remove("active"));
+
+  const selectedContent = document.getElementById(tabName);
+  if (selectedContent) {
+    selectedContent.classList.add("active");
+  }
+
+  if (clickedTab) {
+    clickedTab.classList.add("active");
+  }
+}
+
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
If we add event listeners, it's preferable we have code to remove those event listeners. This may not be completely necessary since it's all isolated to a "static" tab, but if it isn't too hard a change, it might be worth doing